numpy/linalg/linalg.py

```diff
@@ -531,6 +537,40 @@ def inv(a):
            [[-1.25,  0.75],
             [ 0.75, -0.25]]])
 
+    If a matrix is close to singular, the computed inverse may not satisfy
+    ``dot(a, ainv) = dot(ainv, a) = eye(a.shape[0])`` even if a `LinAlgError`
```

Comment:
```suggestion
    ``a @ ainv = ainv @ a = eye(a.shape[0])`` even if a `LinAlgError`
```
(Was going to suggest this, but I think this is better unless we change the text above to use `@` also).

Reply:
Change it all. I think it's better anyways, thanks for flagging!